creating-data-assets/modules/telemetry/telemetry-types-specialized.md

```diff
@@ -0,0 +1,231 @@
+:::
+
+In this telemetry type, angle information is provided for both planets and their moons (if present). The method for calculating these angles varies based on the [Angle Calculation Mode](./angle-information.md) in use. However, the method differs when calculating the angle for the planet versus its moons. For planets, the angle is calculated from the camera to the planet. For moons, the angle is calculated from the planet to the moon while taking the camera's orientation into account. For further details, see {ref}`Moon Angle <moon-angle-horizontal>` for both the {ref}`Horizontal <moon-angle-horizontal>` and {ref}`Circular <moon-angle-circular>` angle calculation modes.
+
+:::{note}
+When the Planets Overview Sonification is enabled, the angle calculated for the planet is modified in this telemetry type. Instead of calculating the angle from the camera to the planet, the angle is calculated similarly to that for the moons, from the Sun to the planet. More information can be found in [Further Notes](#further-notes).
+:::
+
+The OSC messages from this telemetry type are divided among the planets that have been added. These messages are sent under OSC labels such as `/Earth`, `/Mars`, etc. Each message contains at least four items, but there may be more depending on the number of major moons the planet has. Each item is explained in detail below:
+
+  1. The distance from the camera to the planet in kilometers.
+  1. The horizontal angle in radians to the planet, with the current angle calculation mode taken into account. For more information, see [Angle Calculations](./angle-information.md).
+  1. The elevation angle in radians to the planet, with the current angle calculation mode taken into account. Again, see [Angle Calculations](./angle-information.md) for details.
+  1. List of user interface settings for the planet sonification, which aspects of the sonification should be turned on or off. A value of 0 means that it is turned off, and a 1 means that it is turned on. The order of the settings can be seen in the table below. If the setting does not exist for a planet, the value is always 0.
+  1. (optional) The distance from the camera to the **first** moon in kilometers.
+  1. (optional) The horizontal angle in radians to the first moon.
+  1. (optional) The elevation angle in radians to the first moon.
+  1. (optional) The distance from the camera to the **second** moon in kilometers.
+  1. (optional) The horizontal angle in radians to the second moon.
+  1. (optional) The elevation angle in radians to the second moon.
+  1. (optional) ... The data then continues in the same pattern for as many moons as the planet has, with three values per moon. The moons are in the order of distance to the planet, the closest moon first and the farthest away moon last, which was specified in the _planets.asset_ file located in the _data\assets\modules\telemetry\sonification_ folder.
```

Comment:
... ?

Reply:
Yes exactly, there is a variable amount of datapoints in this message depending on the number of moons. But they follow the same pattern.

Reply:
Mh.. I think it confuses more than just leaving out the ellipses

Reply:
Maybe remove both the ellipses and the `optional` part, to make this line a bit more distinguishable from the others?
```suggestion
  1. The data then continues in the same pattern for as many moons as the planet has, with three values per moon. The moons are in the order of distance to the planet, the closest moon first and the farthest away moon last, which was specified in the _planets.asset_ file located in the _data\assets\modules\telemetry\sonification_ folder.
```

![image](https://github.com/user-attachments/assets/9d303e5c-1235-4ed9-b067-f0dce0eb345a)

Reply:
Also, the sentence could be shortened a bit to be easier to read:
```suggestion
  1. The data then continues in the same pattern for each of the planet's moons, with three values per moon. The moons are given in order by distance from the planet (closest first, farthest last) as specified in the _planets.asset_ file in the _data\assets\modules\telemetry\sonification_ folder.
```